packages/yoco/README.md

```diff
@@ -0,0 +1,61 @@
+# Yoco Icon
+
+In-house-designed icons for [kotti design system](https://3yourmind.github.io/kotti/).
+
+[👓 Check the Icons](https://3yourmind.github.io/kotti/foundations/icons/)
+
+## Installation
+
+**Using yarn**
+
+```bash
+yarn add @3yourmind/yoco
+```
+
+**Using npm**
+
+```bash
+npm install --save @3yourmind/yoco
+```
+
+## Add Yoco Icon to your project
+
+**With module bundler**
+
+```scss
+@import '~@3yourmind/yoco'; // might need a sass importer like @3yourmind/sass-node-modules-importer
+```
+
+```typescript
+// enum of icons and the associated ligature / svg name
+import { Icons } "@3yourmind/yoco";
```

Comment:
according to what i saw on the index and the types, it's `Icon` not `Icons`

Reply:
Good catch, this is because I renamed `Icons` to `Icon` at some point IIRC.

Reply:
will resolve and wait for commit then :+1: